sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql

```diff
@@ -2,12 +2,30 @@ CREATE TEMP FUNCTION one_index(x ANY TYPE) AS (
   CAST(IF(SAFE_CAST(x AS INT64) < 0, SAFE_CAST(x AS INT64), SAFE_CAST(x AS INT64) + 1) AS STRING)
 );
 
-CREATE TEMP FUNCTION one_index_struct(x STRUCT<k STRING, v INT64>) AS (
-  STRUCT(one_index(x.k) AS key, x.v AS value)
+CREATE TEMP FUNCTION one_index_struct(record STRUCT<k STRING, v INT64>) AS (
+  STRUCT(one_index(record.k) AS key, record.v AS value)
 );
 
-CREATE TEMP FUNCTION one_index_array(x ARRAY<STRUCT<k STRING, v INT64>>) AS (
-  ARRAY(SELECT one_index_struct(e) FROM UNNEST(x) AS e)
+CREATE TEMP FUNCTION one_index_array(map ARRAY<STRUCT<k STRING, v INT64>>) AS (
+  ARRAY(SELECT one_index_struct(record) FROM UNNEST(map) AS record)
+);
+
+CREATE TEMP FUNCTION filter_probe_counts_with_outlier_values(map ARRAY<STRUCT<k STRING, v INT64>>) AS (
+  -- Similar to search probes/metrics, count values over 10000 are considered too high
+  ARRAY(
+    SELECT
+      STRUCT(record.k AS key, record.v as value)
+    FROM
+      UNNEST(map) AS record
+    WHERE record.v <= 10000
+  )
```

Comment:
RE: https://github.com/mozilla/bigquery-etl/pull/2321#discussion_r705411601 

Filter rather than set to 0 @teonbrooks @akkomar